src/app/tasks/task-ilo-alignment/task-ilo-alignment-viewer/task-ilo-alignment-viewer.tpl.html

```diff
@@ -59,13 +55,7 @@ <h5>Related Tasks</h5>
                 <li class="list-group-item" ng-repeat="align in unit.task_outcome_alignments | outcomeFilter: ilo.id | orderBy: 'seq'">
                   <div class="clearfix">
                     {{taskDef = unit.taskDef(align.task_definition_id); ''}}
-                    <task-ilo-alignment-rater
-                      class="pull-right"
-                      readonly="true"
-                      unit="unit"
-                      show-tooltips="true"
-                      ng-model="align">
-                    </task-ilo-alignment-rater>
+                    <task-ilo-alignment-rater class="pull-right" [unit]="unit" [alignment]="align"></task-ilo-alignment-rater>
```

Comment:
Where is pull-right coming from? Looks like old bootstrap css

Reply:
I focused on the inside of the component, not the outside. 
The pull-right class is defined in task ilo alignment viewer component (scss).
And I think I don't need to adjust it.
Complete section:
![image](https://user-images.githubusercontent.com/31674374/104145065-c9b61100-5419-11eb-869c-0b7af02ceda1.png)